README.md

```diff
@@ -21,6 +21,47 @@ A live-editing time travel environment for Redux.
 npm install --save-dev redux-devtools
 ```
 
+DevTools is a middleware, which should be added to your middleware stack *after* `applyMiddleware` because `applyMiddleware` is potentially asynchronous. Otherwise, DevTools won't see the raw actions emitted by the Promise middleware etc.
```

Comment:
>DevTools is a middleware

It's a “store enhancer”. The result of `applyMiddleware` is also a “store enhancer”.